src/styles/blocks/details.scss

```diff
@@ -0,0 +1,138 @@
+.details {
+  padding: 0 20px 40px;
+  color: var(--secondary-color);
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  font-family: var(--font-family);
+  font-style: normal;
+  margin: 0 auto;
+  max-width: var(--max-width);
+  &_button {
+    padding: 15px 110px;
+    background-color: var(--secondary-color);
+    color: rgba(29, 29, 29, 1);
+    border-radius: 40px;
+    text-decoration: none;
+  }
+}
+
+.details_img {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  gap: var(--box-padding);
+  overflow: hidden;
+}
+
+.details_picture {
+  width: 130px;
+  height: 130px;
+  object-fit: cover;
+}
+
+.details_heading {
+  padding-top: 16px;
+  font-size: 24px;
+  font-weight: 700;
+  line-height: 140%;
+}
+
+.details_paragraf {
+  padding-top: 8px;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 140%;
+}
+
+.button {
+  font-family: var(--font-family);
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 27px;
+  margin: 0 auto;
+  padding: 20px 110px;
+  display: grid;
+  justify-content: center;
+  align-content: center;
+}
+
+.details_title {
+  text-align: center;
+  padding-top: 92px;
+  padding-bottom: 48px;
+  font-size: 32px;
+  font-weight: var(--font-weight);
+  line-height: 32px;
+  letter-spacing: -1.5px;
+  text-transform: capitalize;
+  font-style: normal;
+  color: var(--secondary-color);
+  font-family: var(--font-family);
+}
+
+@media screen and (min-width: 744px) {
+  .details {
+    display: grid;
+    grid-gap: 10px;
+    padding-left: 76px;
+    padding-right: 76px;
+    grid-template-rows: auto;
+  }
+  .details_title {
+    text-align: center;
+    font-size: 48px;
+    font-weight: var(--font-weight);
+    line-height: 100%;
+  }
+  .details_img {
+    grid-template-columns: 2fr 1fr;
+  }
+
+  .details_picture {
+    width: 100%;
+    height: 100%;
+  }
+
+  .details_button {
+    display: flex;
+    width: 392px;
+    height: 56px;
+    padding: 0 48px;
+    justify-content: center;
+    align-items: center;
+  }
+  .details_img--reversed {
+    grid-template-columns: 1fr 2fr;
+  }
+}
+
+@media screen and (min-width: 1220px) {
+  .details_heading {
+    width: 672px;
+  }
+  .details_paragraf {
+    width: 672px;
+    height: 66px;
+
+  }
+  .details_button {
+    width: 276px;
+    &:hover {
+      cursor: pointer;
+      transition: box-shadow 0.3s;
```

Comment:
The transition mustn't be inside a hover. That's why your transitions are not smoothly going back to initial styles when you stop hovering them. Is should be moved up.

Reply:
To be clear - you should do so for all transitions in order to have them nicely 'animated'